src/common/src/config.rs

```diff
@@ -573,25 +567,21 @@ pub struct StorageConfig {
     #[serde(default = "default::storage::max_version_pinning_duration_sec")]
     pub max_version_pinning_duration_sec: u64,
 
-    #[serde(default = "default::storage::compactor_max_sst_key_count")]
-    pub compactor_max_sst_key_count: u64,
-    #[serde(default = "default::storage::compact_iter_recreate_timeout_ms")]
-    pub compact_iter_recreate_timeout_ms: u64,
-    #[serde(default = "default::storage::compactor_max_sst_size")]
-    pub compactor_max_sst_size: u64,
-    #[serde(default = "default::storage::enable_fast_compaction")]
-    pub enable_fast_compaction: bool,
-    #[serde(default = "default::storage::max_preload_io_retry_times")]
-    pub max_preload_io_retry_times: usize,
     #[serde(default, flatten)]
     pub unrecognized: Unrecognized<Self>,
 
+    #[serde(default = "default::storage::max_preload_io_retry_times")]
+    pub max_preload_io_retry_times: usize,
+
     /// The spill threshold for mem table.
     #[serde(default = "default::storage::mem_table_spill_threshold")]
     pub mem_table_spill_threshold: usize,
 
     #[serde(default)]
     pub object_store: ObjectStoreConfig,
+
+    #[serde(default)]
+    pub compactor: CompactorConfig,
```

Comment:
This seems to be a breaking change. For example, if user defines the config previously via:
```
[storage]
compactor_max_task_multiplier=4
```
This config will become invalid and user needs to change it toml to:
```
[storage.compactor]
compactor_max_task_multiplier=4

```

Reply:
revert it